### PR TITLE
Create SDG sanity test for home, countries and regions.

### PR DESCRIPTION
**Pull request by keyurva**
* Does not include SDG_* checks.

### CODE REVIEW

File: server/webdriver/tests/standalone/sdg_sanity.py [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+COUNTRIES: dict[str, dict] = load_countries()
+
+
+class Mode:
+  ALL = "all"
+  HOME = "home"
+  COUNTRY = "country"
+
+
+flags.DEFINE_enum(
```
Comment (pradh) [inline server/webdriver/tests/standalone/sdg_sanity.py:65]: nit: flags right after imports?

File: server/webdriver/tests/standalone/sdg_sanity.py [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+                          expandable, country_dcid, stat_var, e)
+          continue
+
+        # Sleep for 100ms so the elems can expand.
+        time.sleep(0.1)
+        stat_var_elems = find_elems(expandable, By.XPATH,
+                                    CountryConstants.STAT_VAR_XPATH)
+        if stat_var_elems:
+          for stat_var_elem in stat_var_elems:
+            # if the attribute ends with "dc/*", then that is the statvar to navigate to.
```
Comment (pradh) [inline server/webdriver/tests/standalone/sdg_sanity.py:316]: Is the `dc/` true for all SVs?  I thought we may have sdg/ stuff?

**Review by pradh — APPROVED** [review]
Very nice!

File: server/webdriver/tests/standalone/sdg_sanity.py (hunk 2 above)
Comment (keyurva) [reply server/webdriver/tests/standalone/sdg_sanity.py:316]: It seems to be of the form `dc/topic/sdg_*`. e.g. https://datcom-un-sdg.ue.r.appspot.com/UNSDWebsite/undatacommons/sdgs/goals?v=dc%2Ftopic%2Fsdg_6.2.1

File: server/webdriver/tests/standalone/sdg_sanity.py (hunk 1 above)
Comment (keyurva) [reply server/webdriver/tests/standalone/sdg_sanity.py:43]: Done.

**Pull request merged**